src/pudl/workspace/datastore.py

```diff
@@ -491,21 +494,41 @@ def main_arguments():
     )
 
     parser.add_argument(
-        "--dataset", help="Get only a specified dataset. Default gets all.")
+        "--dataset",
+        help="Download the specified dataset only. See below for available options. "
+        "The default is to download all, which takes a long time depending on network "
+        "speed."
```

Comment:
can we just say "which can take a long time" and assume folks know that network speed changes download times? Or better yet just say "The default is to download all"